README.md

```diff
@@ -3,35 +3,23 @@
 ## By: Adam D. Cornett
 
 ## Overview
-AWS Controllers for Kubernetes ("ACK") is an open source project that project that enables a user to define 
-and use AWS resources directly from Kubernetes and OpenShift. Each AWS service, say S3 for example has it's own Operator 
+AWS Controllers for Kubernetes ("ACK") is an open source project that enables a user to define 
+and use AWS resources directly from Kubernetes and OpenShift. Each AWS service, say S3 for example has its own Operator 
 (controller manager), that allows a user to define, create, update, delete an S3 bucket. This is very powerful tool to be 
 able to manage AWS resources, directly from an OpenShift cluster, reducing time/effort to create AWS resources. 
```

Comment:
This section goes back and forth between referring to a specific service (S3) and speaking generally (AWS service). Probably can separate the two. Speak generally, then after give the specific example.